test/index.spec.ts

```diff
@@ -2,22 +2,60 @@ import * as dotenv from 'dotenv';
 import axios from 'axios';
 import { expect } from 'chai';
 import { startServer } from '../src/start-server';
+import { User } from '../src/entity/User';
+import { validateEmail, validatePassword } from '../src/validators/validators';
 
 dotenv.config({ path: './test.env' });
 
 before('Starting server', async () => {
   await startServer();
 });
 
+const connection = axios.create({ baseURL: process.env.APOLLO_SERVER_BASE_URL });
+
 describe('query hello', function () {
-  it('Returning hello', async function () {
+  it('Returning hello', async () => {
     const query = `query { hello }`;
 
-    const connection = axios.create({ baseURL: process.env.APOLLO_SERVER_BASE_URL });
     const result = await connection.post('/graphql', { query });
     const queryResponseField = JSON.stringify(result.data.data.hello);
     console.table(queryResponseField);
 
     expect(queryResponseField).to.be.eq('"Hello, world!"');
   });
 });
+
+describe('Test createUser', () => {
+  it('Returning user', async () => {
+    const query = `mutation($name: String!, $email: String!, $password: String!, $birthdate: String!) {
+      createUser(name: $name, email: $email, password: $password, birthdate: $birthdate) {
```

Comment:
aqui poderia unificar as variáveis em apenas uma, tanto para legibilidade quanto para facilitar adição de novos campos
```suggestion
    const query = `mutation($input: UserInput!) {
      createUser(input: $input) {
```